tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -612,12 +612,30 @@ def test_search_registered_model_order_by(self):
                                                    order_by=['last_updated_timestamp DESC'],
                                                    max_results=100)
         self.assertEqual(rms[::-1], result)
+        # timestamp returns same result as last_updated_timestamp
+        result, _ = self._search_registered_models(query,
+                                                   page_token=None,
+                                                   order_by=['timestamp DESC'],
```

Comment:
for for different flavors of blanks
- space (already covered)
- multiple spaces
- tabs